landing/src/components/modules/filters.tsx

```
@@ -1,18 +1,22 @@
+const frameworksList = Object.values(ResourcesIds)
+  .filter((value) => isNaN(Number(value)))
+  .map((value) => ({
+    value: value.toString(),
+    label: value.toString(),
+  }));
 
 export const AttributeFilters = () => {
-  const [selectedFrameworks, setSelectedFrameworks] = useState<string[]>(["react", "angular"]);
+  const [selectedFrameworks, setSelectedFrameworks] = useState<string[]>(
+    Object.values(ResourcesIds)
+      .filter((value) => isNaN(Number(value)))
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)